### ISSUE TITLE
improve usability of 'Photos' entry

### ISSUE THREAD

**Issue by asaage** [nextcloud/android#3667]
The entry '**Photos**' displays a bucket of thumbnails sorted by upload-date (i guess).
There is no option to reveal any hierarchy or order them differently and it takes ages to display large amounts of thumbnails. This is not very usefull i think.

### solution i'd like
This menu-entry should probably behave more like the nc-gallery-app.
in particular:
- respect ```.nomedia```-files!
- view hierarchically as Album-folders to be able to drill down (that would also reduce the thumbnail-latency i suppose)
- **Bonus**:
  - support ```gallery.cnf``` files
  - pause/play button
  - rename the whole thing to 'Gallery'

**Comment by asaage** [comment]
Apparently this is in fact related to #567 but honestly i can't see the benefit of such a view.

**Comment by Queuecumber** [comment]
The web app has a lot more options for displaying photos, most notably viewing by album, which I consider minimum functionality for a photo viewer. The Android app should at least be brought to feature parity with the web app

**Comment by stucki** [comment]
I would also like to request viewing by album functionality. If there's any way I can support that, let me know.

**Comment by saywebsolutions** [comment]
> I would also like to request viewing by album functionality. If there's any way I can support that, let me know.

yes album functionality would be great. Unfortunately I think there has been some backwards progress; a recent updated renamed the photos section to 'Media' and it is now broken for many users. Hoping for a fix soon and then maybe new features like albums can get some attention

**Comment by AykutCevik** [comment]
>respect .nomedia-files

This is a bug, isn't it? I faced this issue since I saw hidden files in the gallery view of the android app whereas they shouldn't be visible at all.
